### PR TITLE
[#175441170] Convert serviceId to lowercase in logo path

### PR DESCRIPTION
**Pull request by infantesimone**
This PR convert _serviceId_ to lowercase in logo path.

For example, given a _serviceId_ TEST123:
- testcdn.it/logos/services/test123.png

### CODE REVIEW

File: src/pages/LogoOrganizations.tsx [hunk 1]
```diff
@@ -184,7 +184,7 @@ class LogoOrganizations extends Component<Props, LogoState> {
                       logoIsValid
                     }
                     isValid={logoIsValid}
-                    logoPath={`${ORGANIZATIONS_LOGO_PATH}${logoId}`}
+                    logoPath={`${ORGANIZATIONS_LOGO_PATH}${logoId}.png`.toLowerCase()}
```
Comment (AleDore) [inline src/pages/LogoOrganizations.tsx:187]:
```suggestion
                    logoPath={`${ORGANIZATIONS_LOGO_PATH}${logoId.toLowerCase()}.png`}
```

File: src/pages/SubscriptionService.tsx [hunk 2]
```diff
@@ -344,7 +344,9 @@ class SubscriptionService extends Component<Props, SubscriptionServiceState> {
                     errorLogoUpload={errorLogoUpload}
                     isSubmitEnabled={logo !== undefined && logoIsValid}
                     isValid={logoIsValid}
-                    logoPath={`${SERVICES_LOGO_PATH}${service.service_id}`}
+                    logoPath={`${SERVICES_LOGO_PATH}${
+                      service.service_id
+                    }.png`.toLowerCase()}
```
Comment (AleDore) [inline src/pages/SubscriptionService.tsx:349]:
```suggestion
                    logoPath={`${SERVICES_LOGO_PATH}${
                      service.service_id.toLowerCase()
                    }.png`}
```

**Review by AleDore — CHANGES_REQUESTED** [review]
I'd rather move lowercase operation, in order to avoid lowercase to other path fragments

File: src/pages/LogoOrganizations.tsx (hunk 1 above)
Comment (infantesimone) [reply src/pages/LogoOrganizations.tsx:187]: correct

**Review by gunzip — APPROVED** [review]

File: src/pages/LogoOrganizations.tsx (hunk 1 above)
Comment (gunzip) [reply src/pages/LogoOrganizations.tsx:187]: ops, already merged

Comment (infantesimone) [reply src/pages/LogoOrganizations.tsx:187]: done https://github.com/pagopa/io-developer-portal-frontend/pull/150

**Pull request merged**